### PR TITLE
Unit test branch

### PR DESCRIPTION
**Pull request by SaveryIV**
In this PR we are creating tests for methods and instances of classes related to the app

### CODE REVIEW

File: spec/book_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    expect(book).to be_an_instance_of(Book)
+  end
+
+  it 'Should be add as a Rental when the method "add_rental" is call' do
+    book = Book.new('Lord of The Rings', 'Tolkien')
+    person = Person.new(22, 'Austin')
+    result = book.add_rental(person, '2010/07/01')
+    expect(result).to be_an_instance_of(Rental)
+  end
+end
```
Comment (V-Blaze) [inline spec/book_spec.rb:18]: - You have done great so far :+1: :+1: But there are a few requirements you still need to meet before moving to the next milestone. :+1: :100: 

- - 
Selection_016

- [ ] As stated in the project requirement, you need to also write tests for all the `Decorator` classes and methods. Currently, the tests for these have not been implemented. I suggest you implement them. :+1: :smile:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @SaveryIV, @Xghayor  

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- All tests are passing✔
-  Clean and organized code base👋 
-  Correct git flow✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/book_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+require 'rspec'
```
Comment (zeff96) [inline spec/book_spec.rb:1]: - [ OPTIONAL BUT HIGHLY RECOMMENDED ]Hello! 🛠️ When you include require 'rspec' at the top of your Ruby test file, it's a good practice to add the RSpec gem to your project's Gemfile. This approach offers a more convenient way for anyone using your code to set up and run tests. 📦

**Review by zeff96 — APPROVED** [review]
### Status: APPROVED :green_circle: 
Hi @SaveryIV and @Xghayor , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights
✔️ Linters passing
✔️ Correct workflow
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**